src/plugins/workspace/public/components/service_card/workspace_list_card.tsx

```diff
@@ -158,8 +157,8 @@ export const WorkspaceListCard = (props: WorkspaceListCardProps) => {
         ) : (
           <EuiDescriptionList
             type="column"
-            titleProps={{ style: { width: '70%' } }}
-            descriptionProps={{ style: { width: '30%' } }}
+            titleProps={{ style: { width: '60%' } }}
+            descriptionProps={{ style: { width: '40%' } }}
```

Comment:
any specific reason to update the percent?

Reply:
Hi, since the padding is increased a bit. so I think it may align well for the descriptionProps. 
But now it seems that the difference is not very obvious, it will be changed to 70% 30%

Reply:
Hi, since the padding is increased a bit. so I think it may align well for the descriptionProps. 
But now it seems that the difference is not very obvious, it will be changed to 70% 30%

Reply:
Hi, since the padding is increased a bit. so I think it may align well for the descriptionProps. 
But now it seems that the difference is not very obvious, it will be changed to 70% 30%